Dockerfile

```diff
@@ -46,6 +46,7 @@ RUN cd /opt/robotoff/i18n && \
     bash compile.sh
 COPY --chown=off:off robotoff /opt/robotoff/robotoff/
 COPY --chown=off:off data /opt/robotoff/data
+COPY --chown=off:off datasets /opt/robotoff/datasets
```

Comment:
Hum, I don't think we want to copy datasets in the image.

I think there it is better to have a :
```
RUN mkdir -p /opt/robotoff/datasets && chown off:off -R /opt/robotoff/datasets
```